sfge/src/test/java/com/salesforce/rules/usewithsharingondatabaseoperation/SharingPolicySubclassesTest.java

```diff
@@ -36,31 +36,31 @@ public class SharingPolicySubclassesTest extends BaseUseWithSharingOnDatabaseOpe
 
     // PARENT - NO DECLARATION
 
-    @MethodSource("provideAllDatabaseOperations")
+    @MethodSource("provideSelectDatabaseOperations")
```

Comment:
Made [another comment](https://github.com/forcedotcom/sfdx-scanner/pull/1154#discussion_r1300605575) but it was marked as "outdated" so you may not have seen it.

> Changes these tests to only use a subset of the methods in the Database class. They were taking way too long. Full coverage of all methods is in `SharingPolicySimpleTest.java`.

Since I was already updating some tests earlier in this file, I figured I'd update this as well. Let me know if you'd like me to revert.

Reply:
No need to revert, IMO.